ansible/roles/test/templates/pfc_storm_arista.j2

```diff
@@ -1,5 +1,5 @@
 bash
 cd /mnt/flash
-sudo python {{pfc_gen_file}} -p {{(1).__lshift__(pfc_queue_index)}} -t 65535 -n {{pfc_frames_number}} -i {{pfc_fanout_interface | replace("Ethernet", "et")}} -r {{ansible_eth0_ipv4_addr}} &
+sudo python {{pfc_gen_file}} -p {{(1).__lshift__(pfc_queue_index)}} -t 65535 -n {{pfc_frames_number}} -i {{pfc_fanout_interface | replace("Ethernet", "et") | replace("/", "_")}} -r {{ansible_eth0_ipv4_addr}} &
```

Comment:
Can you elaborate the reason of changing et/NN to et_NN?

Reply:
In bash, interfaces are et3_1 instead of eth3/1 for instance.